biogtr/models/embedding.py

```diff
@@ -62,9 +68,16 @@ def __init__(
         self.temperature = temperature
         self.normalize = normalize
         self.scale = scale
+        self.n_points = n_points
+
         if self.normalize and self.scale is None:
             self.scale = 2 * math.pi
 
+        if self.emb_type == "pos" and n_mlp_layers:
+            self.mlp = MLP(self.features * 2, self.features, n_mlp_layers)
```

Comment:
Consider having the MLP module architecture a configurable param (e.g., `n_mlp_features`)

Reply:
done